internal/action/otp.go

```diff
@@ -108,18 +105,30 @@ func (s *Action) otp(ctx context.Context, name, qrf string, clip, pw, recurse bo
 			return nil
 		default:
 		}
-		two, label, err := otp.Calculate(name, sec)
+
+		two, err := otp.Calculate(name, sec)
 		if err != nil {
 			return exit.Error(exit.Unknown, err, "No OTP entry found for %s: %s", name, err)
 		}
-		token := two.OTP()
+
+		token, err := totp.GenerateCodeCustom(two.Secret(), time.Now(), totp.ValidateOpts{
```

Comment:
I guess we might want to switch on:
https://github.com/pquerna/otp/blob/60112ee2a95553a491c22956619ef30260ec93e8/otp.go#L99

if we really want to support both HOTP and TOTP.

Reply:
Done.